x-pack/plugins/data_enhanced/public/search/ui/connected_search_session_indicator/connected_search_session_indicator.test.tsx

```diff
@@ -104,3 +113,151 @@ test('should be disabled during auto-refresh', async () => {
 
   expect(screen.getByTestId('searchSessionIndicator').querySelector('button')).toBeDisabled();
 });
+
+describe('tour steps', () => {
+  describe('loading state', () => {
+    beforeAll(() => {
+      jest.useFakeTimers();
+    });
+
+    afterAll(() => {
+      jest.useRealTimers();
+    });
+
+    test('Shows tour step on loading state with delay', async () => {
+      const state$ = new BehaviorSubject(SearchSessionState.Loading);
+      const SearchSessionIndicator = createConnectedSearchSessionIndicator({
+        sessionService: { ...sessionService, state$ },
+        application: coreStart.application,
+        timeFilter,
+        storage,
+      });
+      const rendered = render(<SearchSessionIndicator />);
+
+      await waitFor(() => rendered.getByTestId('searchSessionIndicator'));
+
+      act(() => {
+        jest.advanceTimersByTime(5000);
+      });
+
+      expect(rendered.queryByTestId('searchSessionIndicatorTour')).toBeTruthy();
+
+      act(() => {
+        jest.advanceTimersByTime(5000);
+        state$.next(SearchSessionState.Completed);
+      });
+
+      // Open tour should stay on screen after state change
```

Comment:
> Open tour should stay on screen after state change

Just thinking out loud: 
this was a bit confusing for me while testing:
The first tour appeared and while I was reading the text state has changed and the explanation isn't accurate for a new state: 
For example, I can't cancel in this new state: 

![Screenshot 2021-01-26 at 15 35 28](https://user-images.githubusercontent.com/7784120/105858876-20ac1080-5fec-11eb-9c6f-6404b7aad55b.png)

Reply:
This is a bug in my implementation.
Once the tour is open, it should never change.
Will fix and add a test to verify.

Reply:
I think the wording needs to be improved (maybe not address the cancellation altogether)